lib/modules/platform/gitea/index.md

```diff
@@ -17,8 +17,13 @@ You must set `platform=gitea` in your Renovate config file.
 
 The PAT should have these permissions:
 
-- `repo`
-- `read:user` and `read:email`
+| Permission  | Valid for Gitea versions |
+| ----------- | ----------- |
+| `repo`      | all       |
+| `read:user` | all        |
+| `issue`     | `>= 1.20.0`|
+| `read:email`| `<= 1.19.3`|
+| `read:misc` | Only for `1.20.0` and `1.20.1`|
```

Comment:
```suggestion
| Permission   | Valid for Gitea versions       |
| ------------ | ------------------------------ |
| `repo`       | all                            |
| `read:user`  | all                            |
| `issue`      | `>= 1.20.0`                    |
| `read:email` | `<= 1.19.3`                    |
| `read:misc`  | Only for `1.20.0` and `1.20.1` |
```

Apply this suggestion to fix the Prettier lint test.

If you have a development environment setup, or use GitHub Codespaces, you can also run `yarn prettier-fix` and commit the fixed layout, and push it to your branch.